lib/DDG/Spice/IndegoBikeshareLocations.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::IndegoBikeshareLocations;
```

Comment:
I agree with @marianosimone City + Systemname seems best, e.g. PHLIndego for Philadelphia